src/helpers/importedFiles.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const path = require('path');
+
+const BemCell = require('@bem/sdk.cell');
+
+const cellPath = require('./cellPath');
+const namingConvention = require('./namingConvention');
+
+// TODO: suggest more elegant solution
+//   a. keeping full path in config
+//   b. allowing to build short bem-cell path without level part
+function resolveCellPath(relativePath, levelPath) {
+    return path.resolve(levelPath, '../' + relativePath);
+}
+
+/**
+ * Returns paths for BEM-entities across all levels
+ */
+module.exports = (entities, levels, { techs, techMap }) => {
+    let bemDeps = [];
+
+    entities.forEach(entity => {
+        const entityTechs = entity.tech ? [entity.tech] : techs;
+
+        entityTechs.forEach(tech => {
+            const techExt = techMap[tech] || tech;
+            const files = levels.map(level => {
+                const convention = namingConvention.getFromLevel(level);
+                const cell = BemCell.create({ entity, tech: techExt, layer: level.layer });
+
+                return {
+                    path: resolveCellPath(cellPath(cell, convention), level.path),
+                    tech
+                };
+            });
+
+            // TODO: should we add __elem (in case of __elem_mod) and _mod (in case of _mod_val) ?
+            //   like webpack-bem-loader does
+
+            bemDeps = bemDeps.concat(files);
+        });
+    });
+
+    // FIXME: does it have right order ?
```

Comment:
Guess we can't reorder this anyway. Or you know how we can?

Reply:
Yes its rhetorical somehow. I'll remove it